Documentation/DownloadingTheMRTK.md

```diff
@@ -0,0 +1,36 @@
+# How to download the Mixed Reality Toolkit
+
+![](/External/ReadMeImages/MRTK_Logo_Rev.png)
+
+Recognizing there are multiple ways in which developers want to be able to get access to project through Unity, the Mixed Reality Toolkit has been engineered to be available through the following methods:
+
+## Unity Asset downloadable from the MRTK GitHub site
+
+You can download the packaged [Unity Asset from here](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases) for importing in to your project.
+
+## Unity NuGet delivery (*Coming soon*)
+
+Unity now has an asset plugin for getting access to Unity projects using the dependency delivery solution known as NuGet.
+
+Simply install the [Unity NuGet package](https://assetstore.unity.com/packages/tools/utilities/nuget-for-unity-104640) and search for the Mixed Reality Toolkit to download the project.
+
+## Unity Package Manager (*Coming soon*)
+
+As soon as it's available, the MRTK team will be working with Unity to publish the Mixed Reality Toolkit as a package through their Unity Package Manager.
+
+## Unity Asset Store (*under discussion*)
+
+It has been raised within the Mixed Reality Toolkit team that the package should be published to the Unity store.  Microsoft are in talks with Unity as to the best delivery options.  Likely this will be the Unity Package Manager.
+Feedback is welcome through the [Mixed Reality GitHub site](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) (raise an issue to offer an opinion) and the [HoloDevelopers Slack channel](https://join.slack.com/t/holodevelopers/shared_invite/enQtNDIwNTA4MzIyNDk4LTM3NmU3MTBlMWM4OGU0MGI5YmZmMzhiZmRkZDMxMDZhNzVhNTczYThiYTNhMmEyMTQ2MzA0ZDZhZTY1ZTU2OGQ) for the Mixed Reality Toolkit.
```

Comment:
I think that link is incorrect? 

Shouldn't the long hash at the end of the url be unique per user?

Just checking

Reply:
It was the generic invite link I got from Slack.

Reply:
I think we recently added this type of link with the invite to the master and dev main pages. 

Maybe use that link?

Reply:
Updated